packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -369,20 +369,38 @@ export class StripeHelper extends StripeHelperBase {
   /**
    * Create a stripe customer.
    */
-  async createPlainCustomer(
-    uid: string,
-    email: string,
-    displayName: string,
-    idempotencyKey: string,
-    ipAddress?: string
-  ): Promise<Stripe.Customer> {
+  async createPlainCustomer(args: {
+    uid: string;
+    email: string;
+    displayName: string;
+    idempotencyKey: string;
+    countryCode?: string;
+    postalCode?: string;
+  }): Promise<Stripe.Customer> {
+    const { uid, email, displayName, idempotencyKey, countryCode, postalCode } =
+      args;
+
+    const shipping =
+      countryCode && postalCode
+        ? {
+            name: email,
+            address: {
+              country: countryCode,
+              postal_code: postalCode,
+            },
+          }
+        : undefined;
```

Comment:
Would it be useful to track when `countryCode` and `postalCode` could not be determined, and for what IP address?

Reply:
It could be - how do we want to log this? On the Stripe customer? Logger?

Reply:
Updated buildTaxAddress to log when it's unable to match an IP address to a tax location.